src/components/AppFooter.js

```diff
@@ -53,8 +53,8 @@ export default function AppFooter() {
         <div className="footerBottomTitle">Open Sacramento</div>
         <div className="footerBottomLinksContainer">
           <Link href="/get-started" className="footerBottomLink">get started</Link>
-          <Link href="/" className="footerBottomLink">our team</Link>
-          <Link href="/" className="footerBottomLink">code of conduct</Link>
+          <Link href="about/#team" className="footerBottomLink">our team</Link>
```

Comment:
should this be an absolute path?

Reply:
Absolutely, thank you for catching that!